.github/dependabot.yml

```diff
@@ -5,7 +5,11 @@ updates:
     directory: '/'
     labels: ["A1-insubstantial", "R0-silent"]
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
We can probably do the same for the rust deps.

Reply:
done